<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QMainWindow" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>268</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="8">
     <widget class="QPushButton" name="ButtonBack">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: rgb(255, 173, 105);
	padding: 5px;
	border: 1px solid gray;
	border-radius: 5px
}

QPushButton:pressed{
	background-color: #C0C0C0;
	padding: 5px;
	border: 1px solid gray;
	border-radius: 5px
}</string>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </item>
    <item row="7" column="4">
     <widget class="QPushButton" name="ButtonSqrt">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #E1E1E1;
	padding: 5px;
	border: 1px solid gray;
	border-radius: 5px;
}

QPushButton:pressed{
	background-color: #C0C0C0;
	padding: 5px;
	border: 1px solid gray;
	border-radius: 5px;
}</string>
      </property>
      <property name="text">
       <string>sqrt</string>
      </property>
     </widget>
    </item>
    <item row="9" column="4">
     <widget class="QPushButton" name="ButtonChangeSign">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #E1E1E1;
	padding: 5px;
	border: 1px solid gray;
	border-radius: 5px;
}

QPushButton:pressed{
	background-color: #C0C0C0;
	padding: 5px;
	border: 1px solid gray;
	border-radius: 5px;
}</string>
      </property>
      <property name="text">
       <string>+/-</string>
      </property>
     </widget>
    </item>
    <item row="7" column="5">
     <widget class="QPushButton" name="ButtonSubtract">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: white;
	padding: 5px;
	border: 1px solid gray;
	border-radius: 5px;
}

QPushButton:pressed{
	background-color: #C0C0C0;
	padding: 5px;
	border: 1px solid gray;
	border-radius: 5px;
}</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="8" column="7">
     <widget class="QPushButton" name="Button2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: white;
	padding: 5px;
	border: 1px solid gray;
	border-radius: 5px;
}

QPushButton:pressed{
	background-color: #C0C0C0;
	padding: 5px;
	border: 1px solid gray;
	border-radius: 5px;
}</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="5" column="7">
     <widget class="QPushButton" name="ButtonTag">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #E1E1E1;
	padding: 5px;
	border: 1px solid gray;
	border-radius: 5px;
}

QPushButton:pressed{
	background-color: #C0C0C0;
	padding: 5px;
	border: 1px solid gray;
	border-radius: 5px;
}</string>
      </property>
      <property name="text">
       <string>tag(x)</string>
      </property>
     </widget>
    </item>
    <item row="8" column="4">
     <widget class="QPushButton" name="ButtonReciprocal">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #E1E1E1;
	padding: 5px;
	border: 1px solid gray;
	border-radius: 5px;
}

QPushButton:pressed{
	background-color: #C0C0C0;
	padding: 5px;
	border: 1px solid gray;
	border-radius: 5px;
}</string>
      </property>
      <property name="text">
       <string>1/x</string>
      </property>
     </widget>
    </item>
    <item row="9" column="6">
     <widget class="QPushButton" name="Button0">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: white;
	padding: 5px;
	border: 1px solid gray;
	border-radius: 5px;
}

QPushButton:pressed{
	background-color: #C0C0C0;
	padding: 5px;
	border: 1px solid gray;
	border-radius: 5px;
}</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="6" column="7">
     <widget class="QPushButton" name="Button8">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: white;
	padding: 5px;
	border: 1px solid gray;
	border-radius: 5px;
}

QPushButton:pressed{
	background-color: #C0C0C0;
	padding: 5px;
	border: 1px solid gray;
	border-radius: 5px;
}</string>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="9" column="5">
     <widget class="QPushButton" name="ButtonDivide">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: white;
	padding: 5px;
	border: 1px solid gray;
	border-radius: 5px;
}

QPushButton:pressed{
	background-color: #C0C0C0;
	padding: 5px;
	border: 1px solid gray;
	border-radius: 5px;
}</string>
      </property>
      <property name="text">
       <string>รท</string>
      </property>
     </widget>
    </item>
    <item row="8" column="6">
     <widget class="QPushButton" name="Button1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: white;
	padding: 5px;
	border: 1px solid gray;
	border-radius: 5px;
}

QPushButton:pressed{
	background-color: #C0C0C0;
	padding: 5px;
	border: 1px solid gray;
	border-radius: 5px;
}</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="9" column="8">
     <widget class="QPushButton" name="ButtonEqual">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: white;
	padding: 5px;
	border: 1px solid gray;
	border-radius: 5px;
}

QPushButton:pressed{
	background-color: #C0C0C0;
	padding: 5px;
	border: 1px solid gray;
	border-radius: 5px;
}</string>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
    <item row="5" column="5">
     <widget class="QPushButton" name="ButtonSin">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #E1E1E1;
	padding: 5px;
	border: 1px solid gray;
	border-radius: 5px;
}

QPushButton:pressed{
	background-color: #C0C0C0;
	padding: 5px;
	border: 1px solid gray;
	border-radius: 5px;
}</string>
      </property>
      <property name="text">
       <string>sin(x)</string>
      </property>
     </widget>
    </item>
    <item row="5" column="6">
     <widget class="QPushButton" name="ButtonCos">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #E1E1E1;
	padding: 5px;
	border: 1px solid gray;
	border-radius: 5px;
}

QPushButton:pressed{
	background-color: #C0C0C0;
	padding: 5px;
	border: 1px solid gray;
	border-radius: 5px;
}</string>
      </property>
      <property name="text">
       <string>cos(x)</string>
      </property>
     </widget>
    </item>
    <item row="7" column="7">
     <widget class="QPushButton" name="Button5">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: white;
	padding: 5px;
	border: 1px solid gray;
	border-radius: 5px;
}

QPushButton:pressed{
	background-color: #C0C0C0;
	padding: 5px;
	border: 1px solid gray;
	border-radius: 5px;
}</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="7" column="8">
     <widget class="QPushButton" name="Button6">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: white;
	padding: 5px;
	border: 1px solid gray;
	border-radius: 5px;
}

QPushButton:pressed{
	background-color: #C0C0C0;
	padding: 5px;
	border: 1px solid gray;
	border-radius: 5px;
}</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="8" column="8">
     <widget class="QPushButton" name="Button3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: white;
	padding: 5px;
	border: 1px solid gray;
	border-radius: 5px;
}

QPushButton:pressed{
	background-color: #C0C0C0;
	padding: 5px;
	border: 1px solid gray;
	border-radius: 5px;
}</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="8" column="5">
     <widget class="QPushButton" name="ButtonMultiply">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: white;
	padding: 5px;
	border: 1px solid gray;
	border-radius: 5px;
}

QPushButton:pressed{
	background-color: #C0C0C0;
	padding: 5px;
	border: 1px solid gray;
	border-radius: 5px;
}</string>
      </property>
      <property name="text">
       <string>x</string>
      </property>
     </widget>
    </item>
    <item row="6" column="4">
     <widget class="QPushButton" name="ButtonSquare">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #E1E1E1;
	padding: 5px;
	border: 1px solid gray;
	border-radius: 5px;
}

QPushButton:pressed{
	background-color: #C0C0C0;
	padding: 5px;
	border: 1px solid gray;
	border-radius: 5px;
}</string>
      </property>
      <property name="text">
       <string>x2</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4" colspan="5">
     <widget class="QLineEdit" name="screen">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
	background-color: #FFFFFF;
	color: #000000;
	border: 1px solid gray;
}</string>
      </property>
      <property name="text">
       <string>0.0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="9" column="7">
     <widget class="QPushButton" name="ButtonPoint">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: white;
	padding: 5px;
	border: 1px solid gray;
	border-radius: 5px;
}

QPushButton:pressed{
	background-color: #C0C0C0;
	padding: 5px;
	border: 1px solid gray;
	border-radius: 5px;
}</string>
      </property>
      <property name="text">
       <string>.</string>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QPushButton" name="ButtonClear">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: rgb(255, 173, 105);
	padding: 5px;
	border: 1px solid gray;
	border-radius: 5px
}

QPushButton:pressed{
	background-color: red;
	padding: 5px;
	border: 1px solid gray;
	border-radius: 5px
}</string>
      </property>
      <property name="text">
       <string>AC</string>
      </property>
     </widget>
    </item>
    <item row="6" column="6">
     <widget class="QPushButton" name="Button7">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: white;
	padding: 5px;
	border: 1px solid gray;
	border-radius: 5px;
}

QPushButton:pressed{
	background-color: #C0C0C0;
	padding: 5px;
	border: 1px solid gray;
	border-radius: 5px;
}</string>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="7" column="6">
     <widget class="QPushButton" name="Button4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: white;
	padding: 5px;
	border: 1px solid gray;
	border-radius: 5px;
}

QPushButton:pressed{
	background-color: #C0C0C0;
	padding: 5px;
	border: 1px solid gray;
	border-radius: 5px;
}</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="6" column="8">
     <widget class="QPushButton" name="Button9">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: white;
	padding: 5px;
	border: 1px solid gray;
	border-radius: 5px;
}

QPushButton:pressed{
	background-color: #C0C0C0;
	padding: 5px;
	border: 1px solid gray;
	border-radius: 5px;
}</string>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="6" column="5">
     <widget class="QPushButton" name="ButtonAdd">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: white;
	padding: 5px;
	border: 1px solid gray;
	border-radius: 5px;
}

QPushButton:pressed{
	background-color: #C0C0C0;
	padding: 5px;
	border: 1px solid gray;
	border-radius: 5px;
}</string>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>423</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
